LjinuxRoot/bin/jpkg.py

```diff
@@ -0,0 +1,184 @@
+opts2 = ljinux.based.fn.xarg()
+errored = False
+
+jpkg_version = 0
+
+if len(opts2["w"]) > 1 and opts2["w"][0] == "install":
+    # preperation
+
+    fl = []  # folder list
+
+    # load system package list
+    ljinux.based.command.fpexecc([None, "/etc/jpkg/tools/generatelist.py"])
+    pklist = ljinux.based.user_vars["return"]
+    """
+    pklist[0] is packages w/ version, deps, conficts
+    pklist[1] is a set of all deps
+    pklist[2] is a set of all conflicts
+    """
+    ljinux.based.user_vars["return"] = "1"
+    # we do not want to leave the whole package list in return
+
+    # extraction
+    for fileext in opts2["w"][1:]:
+        if errored:
+            print("JPKG Error: A fatal error has occured, exiting..")
+            break
+        ljinux.based.silent = True
+        ljinux.based.command.fpexecc([None, "/bin/random.py"])
+        ljinux.based.silent = False
+        extpath = "/tmp/" + ljinux.based.user_vars["return"][2:] + fileext[:-4]
+        ljinux.based.user_vars["argj"] = "a " + extpath
+        ljinux.based.command.fpexecc([None, "/bin/mkdir.py"])
+
+        print(f"JPKG: Extracting {fileext[:-4]} ...")
+        ljinux.based.user_vars["argj"] = f"a -q -d {fileext} {extpath}"
+        ljinux.based.command.fpexecc([None, "/bin/jz.py"])
+        if ljinux.based.user_vars["return"] == "0":
+            print(f"JPKG: Extracted to " + extpath)
```

Comment:
Warning: Use of print instead of system.write()

Reply:
`\n` is desired in these that `f` there is sus tho
`stdout.write()` is only benefitial when we want to skip print's logic and just print raw